.buildkite/pipeline.yml

```diff
@@ -15,6 +15,7 @@ steps:
   - label: "Init environment :computer:"
     key: "init_cpu_env"
     command:
+      - "julia --project=test -e 'using Pkg; Pkg.add(url=\"https://github.com/CliMA/CloudMicrophysics.jl\", rev=\"main\")'"
```

Comment:
The GHA unit tests worn't work, because I don't know how to point them to CloudMicrophysics main branch. But hopefully with this we can at least test the builkite jobs.

For testing things locally something like this `add CloudMicrophysics#main` should work in the package manager

Reply:
We did release CloudMicrophysics v0.15. So this is not needed